### ISSUE TITLE
Quiz/surveys not received!

### ISSUE THREAD

**Issue by sistem** [QuizandSurveyMaster/quiz_master_next#424]
Hi guys,

I love this plug-in but lately I'm having some serious issues: several website users of my website told me they completed and sent the test, but we have never received them!
I figured out 2 different problems:
- I actually see in the DB tests which have been completed, but we have never received them by email
- some users say they have filled the test, but we haven't received them and they are not even stored in the DB. 

Anything I can do to solve these problems?
Thanks in advance for your support!

Best
Emanuele

**Comment by dukeran** [comment]
Hey Emanuele,

I received your support ticket and when I tried to email you back, I was notified of an error with the email system. 
Would you be able to test the the email of your WordPress installation? To do so, please install the plugin called Check Email by Chris Taylor which you can find here https://wordpress.org/plugins/check-email/ or you can search for it from your Add New page in your Plugins menu. Once installed, the plugin will add a new Check Email page in the Tools menu. Please go to that page, enter in your email address in the "Send test email to" field and then click "Send Test Email" Leave the other settings at the default. Please let us know if you receive this test email. Once finished, feel free to de-activate and delete the Check Email plugin.

Cheers
Kerrie

**Comment by sistem** [comment]
Hi Kerrie,
Thanks for your support and sorry for the late reply.OK I've have installed the plugin and checked the email, which was correctly received.I also received your email sent to the wordpress admin address (info@romalingue.it) so I don't know why you were notified an error.As I wrote, I noticed 2 different issues: - tests which are stored in the DB, but were not received by mail: it could be either a plugin error or a mail account error - tests which haven't been received and are not even stored in the DB: I don't have any real evidence of these, but more than one users advised us they had filled the test, but we haven't received anything from them. Investigating this issue I found out the first one.
Is it anything already happened to other users? Anything I can do to analyze/solve the issue?
Thanks a lot,Emanuele
